include/criterion/theories.h

```diff
@@ -86,31 +89,201 @@ CR_END_C_API
  */
 # define DataPoints(Type, ...) CR_EXPAND(CR_TH_INTERNAL_DP(Type, __VA_ARGS__))
 
-// Theory invariants
+/**@}*/
+
+/**
+ * @defgroup TheoryInvariants Theory invariants
+ * @{
+ */
 
+/**
+ * Assumes \c Condition is true
+ *
+ * Call: \c cr_assume(Condition).
```

Comment:
This line is unecessary for all doxygen comments, as the macro definition matches the API prototype.